Automattic/wp-calypso
# Customer Home: Migrate Images to Webpack

#### Changes proposed in this Pull Request

This migrates the images of the Customer Home section to Webpack. For details on the reasoning behind this, see #36850.

Note that the Happiness Illustration will still use old colours since it's a .png that wasn't touched when the rest of the illustrations were. I think that's out of scope for this PR.

#### Testing instructions

Verify that all the images load on the Customer Home page: `/home/site?d=launched`. 

cc @eliorivero, @davemart-in 

<img width="1145" alt="Screenshot 2019-12-23 at 08 15 59" src="https://user-images.githubusercontent.com/43215253/71345274-c923b180-255c-11ea-822e-d91b39bc6510.png">
